nntrainer/models/neuralnet.cpp

```diff
@@ -301,6 +301,18 @@ sharedConstTensors NeuralNetwork::forwarding(sharedConstTensors input,
   auto &last_layer =
     model_graph.getSortedLayerNode(model_graph.getSorted().size() - 1).layer;
 
+  for (auto &layer_node : model_graph.getSorted()) {
+    auto l = layer_node.layer;
+    if (l->getType() == "centroid_nn") {
+      if (!label.empty()) {
+        throw std::invalid_argument(
```

Comment:
In this case, let the layer throw than NeuralNetwork throwing.

Reply:
Updated.